ci-operator/config/redhat-developer/helm/OWNERS

```diff
@@ -0,0 +1,18 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- otaviof
+- sbose78
+options: {}
+reviewers:
+- akashshinde
+- akram
+- avni-sharma
+- baijum
+- bkundu
```

Comment:
@ldimaggi I believe Bama's github account is @bamachrn, isn't it?

Reply:
Thanks!